docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: "3.5"
```

Comment:
is it python version?

Reply:
It's the Compose file format version, see https://docs.docker.com/compose/compose-file/

Reply:
It's the Compose file format version, see https://docs.docker.com/compose/compose-file/